crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -6,9 +6,10 @@ use.$kernel::account_id
 use.$kernel::asset_vault
 use.$kernel::constants
 use.$kernel::faucet
-use.$kernel::input_notes
+use.$kernel::input_note
 use.$kernel::memory
 use.$kernel::note
+use.$kernel::output_note
```

Comment:
I think there is an argument for keeping `input_notes` and `output_notes` plural. Most other modules are about working with a single object (e.g., `note` is for working a the current note, `account` is for working with a current account etc.). `input_notes` and `output_notes` are about collections of notes and also every procedure there takes a note ID as a parameter.

So, my slight preference would be to keep these plural - but I also understand the argument for making them singular.

Reply:
Makes sense - let's keep them as is then.